src/System.Private.ServiceModel/tools/scripts/OpenFirewallPorts.cmd

```diff
@@ -0,0 +1,18 @@
+@echo off 
+setlocal
+
+REM 
+REM Start Open all ports WCF test services use
+REM
+
```

Comment:
For the elevation check, I will add it AFTER I enable checking exit code for the OpenFireWallPorts script.